classes/template/PKPTemplateManager.inc.php

```diff
@@ -190,6 +192,22 @@ function initialize() {
 				);
 			}
 
+			// Register color picker assets on the appearance page
+			$this->addJavaScript(
+				'spectrum',
+				Request::getBaseUrl() . '/lib/pkp/js/lib/jquery/plugins/spectrum/spectrum.js',
```

Comment:
Perfect! I probably need to spend some time digging into the `Request`, `Router` and `Dispatcher` classes.